kernel-abi-check/src/python_abi/mod.rs

```diff
@@ -57,13 +57,22 @@ pub static PYTHON_STABLE_ABI: Lazy<HashMap<String, AbiInfo>> = Lazy::new(|| {
     symbols
 });
 
+/// Python ABI violation.
+#[derive(Debug, Clone, Eq, Ord, PartialEq, PartialOrd)]
+pub enum PythonAbiViolation {
+    /// Symbol is newer than the specified Python ABI version.
+    NewerSymbol { name: String, added: Version },
```

Comment:
nit: maybe `UnsupportedVersionSymbol` or `IncompatibleSymbol` more accurately describe the violation?

Reply:
Good idea, I renamed it to `IncompatibleAbi3Symbol` to emphasize that it's still a valid ABI3 symbol.